WordPress/wordpress-playground
# Improve Playground CLI logging and fix quiet mode

## Motivation for the change, related issues

When an error occurs during the boot of Playground CLI users need to understand what happened. 

Currently, a JS error is shown, but there are no PHP logs that commonly explain the root cause.

This PR fixes the quiet mode and adds a debug mode showing PHP logs after boot crashes.

## Implementation details

All log messages are now printed with the logger. This fixes `quiet` mode.

If `debug` is enabled PHP logs will be fetched after a boot crash and thrown together with the JS error. 

## Testing Instructions (or ideally a Blueprint)

- Add this blueprint file to your machine
```
{
    "plugins": [
        "wordfence"
    ]
}
```
- Start the server (replace BLUEPRINT_PATH with your path)
```
bun packages/playground/cli/src/cli.ts server --wp=6.5 --port=9401 --quiet --blueprint=BLUEPRINT_PATH --debug
```
- Confirm that only errors were printed and that PHP errors are included
